.github/workflows/functional-tests.yml

```diff
@@ -723,7 +666,7 @@ jobs:
         run: |
           . ${GITHUB_WORKSPACE}/ci-environment.sh
           cd ${GITHUB_WORKSPACE}/shakenfist
-          tools/run_remote ${primary} sudo /home/${baseuser}/tools/ci_event_checks.sh
+          tools/run_remote ${primary} "sudo bash tools/ci_log_checks.sh develop"
```

Comment:
This is wrong. Upgrade tests have a different list of errors to forbid, specifically around online upgrade. This got lost in 4c8ab68c5f4f9c36cbb873f587e51e1235bd6288 when this was replaced by the wrong script by accident. Before the forbidden list for upgrades was:

```
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 108)           FORBIDDEN=("Traceback (most recent call last):"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 109)                      "ERROR sf"
25b58e380 (Michael Still  2023-08-28 12:06:44 +1000 110)                      "ERROR gunicorn"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 111)                      " died"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 112)                      "Extra vxlan present"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 113)                      "Fork support is only compatible with the epoll1 and poll polling strategies"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 114)                      "not using configured address"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 115)                      "Dumping thread traces"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 116)                      "because it is leased to"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 117)                      "not committing online upgrade"
b1b342208 (Michael Still  2023-05-02 16:02:50 +1000 118)                      "Received a GOAWAY with error code ENHANCE_YOUR_CALM"
4cc396b22 (Michael Still  2023-07-11 10:57:21 +1000 119)                      "ConnectionFailedError"
2e6432a70 (Michael Still  2023-09-09 16:05:59 +1000 120)                      "invalid JWT in Authorization header"
0a0b8abb9 (Michael Still  2023-10-26 10:45:17 +1100 121)                      "Libvirt Error: XML error"
c1b1c0cea (Michael Still  2023-10-29 19:25:30 +1100 122)                      "Cleaning up leaked IPAM"
81d73b038 (Michael Still  2023-11-29 15:36:44 +1100 123)                      "Cleaning up leaked vxlan"
4ad38846f (Michael Still  2023-12-09 18:59:08 +1100 124)                      "Waiting to acquire lock"
4ad38846f (Michael Still  2023-12-09 18:59:08 +1100 125)                      'apparmor="DENIED"'
4ad38846f (Michael Still  2023-12-09 18:59:08 +1100 126)                      "Ignoring malformed cache entry")
```